spec/acceptance/zz281_spec.rb

```diff
@@ -164,9 +164,9 @@ class { 'erlang': epel_enable => true}
     describe file('/var/lib/rabbitmq/rabbitmqadmin') do
       it { should be_file }
     end
-
-    describe command('rabbitmqadmin --help') do
-      it { should return_exit_status 0 }
+    
+    describe command('/usr/local/bin/rabbitmqadmin --help') do
```

Comment:
Hi @cmurphy When I ran the beaker test prior to the change I made and looked on the centos-64-x64 VM, /usr/local/bin was not on the PATH for the root user.

I had a quick look in the module to see where rabbitmqadmin would be called and if we take the [rabbitmq_exchange provider] (https://github.com/puppetlabs/puppetlabs-rabbitmq/blob/master/lib/puppet/provider/rabbitmq_exchange/rabbitmqadmin.rb) as an example,  this has rabbitmqadmin with the full path:

```ruby
if Puppet::PUPPETVERSION.to_f < 3
    commands :rabbitmqctl   => 'rabbitmqctl'
    commands :rabbitmqadmin => '/usr/local/bin/rabbitmqadmin'
  else
    has_command(:rabbitmqctl, 'rabbitmqctl') do
      environment :HOME => "/tmp"
    end
    has_command(:rabbitmqadmin, '/usr/local/bin/rabbitmqadmin') do
      environment :HOME => "/tmp"
    end
  end
```

In my environment (CentOS 6.5) where I am installing rabbitmq via puppet I made a local change to the rabbitmq_exchange provider code and removed the path from the above code. When I ran puppet it failed and was unable to create my test exchange. Reverting back to including the full path and the puppet run was successful.